.travis.yml

```diff
@@ -1,4 +1,17 @@
 language: generic
-before_install: rvm reinstall ruby-head --binary
+addons:
+  apt:
+    packages:
+      - python3
+before_install:
+  - rvm reinstall ruby-head --binary
+  - jdk_switcher use oraclejdk8
+  - ./.travis/install_from_source
 install: gem install jekyll
-script: jekyll build
+before_script: rm ./googlecfd75a4bac031411.html
+script: 
+  - jekyll serve --detach --no-watch
```

Comment:
The install is in an external script, would it make sense to extract this to an external script as well? Maybe even the same one and have flags? eg:
```
before_install:
  - ./.travis/html_validator INSTALL
:
script:
  - ./.travis/html_validator RUN
```
Please feel free to take or leave the above suggestion. I can see the suggestion as being overkill, but finding some way to make this a bit more user friendly for maintenance would be nice.